dotnet-authserver/src/TeacherIdentity.AuthServer/AuthenticationState.cs

```diff
@@ -221,14 +221,20 @@ public void OnEmailSet(string email)
         EmailAddressVerified = false;
     }
 
-    public void OnEmailVerified(User? user)
+    public void OnEmailVerified()
     {
         if (EmailAddress is null)
         {
             throw new InvalidOperationException($"{nameof(EmailAddress)} is not known.");
         }
 
         EmailAddressVerified = true;
+    }
+
+    public void OnEmailVerified(User? user)
```

Comment:
Wanted to assign a default value here, but not sure about the consequences of setting FirstTimeSignInForEmail in the registration journey

Reply:
`FirstTimeSignInForEmail` affects the confirmation screen that a user sees when they've logged in for the outgoing journey. It probably doesn't matter for now whether it's set or not.

Reply:
Yeah I saw it where it was used, I just wasn't sure whether it made sense to set it or not. I'll go with
`public void OnEmailVerified(User? user = null)`
then, and we can change later depending on the designs